packages/FSharp.Formatting/FSharp.Formatting.fsx

```diff
@@ -1,76 +0,0 @@
-#nowarn "211"
```

Comment:
Yep, it's there for bootstrapping reasons. Without this the build fails for multiple reasons:
- The file is no longer found when creating the nuget package
- The bootstrapping sanity check will no longer work

For now the build fails at an earlier point...